Rename Remotion to Multiapp

rtharston opened the pull request:
We've renamed Remotion to Multi, but multi is already taken so I used multiapp, referring to our website Multi.app.

I removed the Remotion cask because it points to an old version we no longer support and that cask can't be updated anymore. If there is a preferred way to deprecate casks, please let me know.


Verified:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:


- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- **NOTE: The token rules say to remove app but there is already a cask named Multi, so I used 'multiapp' because our website is multi.app.** 
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

rtharston commented:
Is it possible to somehow redirect from Remotion to Multiapp instead of just removing Remotion? Or at least leave a note on that cask that it is no longer available and people should move to Multiapp?

bevanjkay commented:
@rtharston I tried to push the change to your branch but am unable to edit. In future please open PRs from a new branch on your repository fork, an ensure that maintainers can push to your PR.

To create the rename - add the change to https://github.com/Homebrew/homebrew-cask/blob/master/cask_renames.json

rtharston commented:
> @rtharston I tried to push the change to your branch but am unable to edit. In future please open PRs from a new branch on your repository fork, 

Thanks. This is the first time I've done this, so I just used `brew create --cask <url> --tap <remotionco/cask>` and let that do its thing. (I also noticed it made the cask in the root folder and I had to manually move the cask to the `m` folder.)

> an ensure that maintainers can push to your PR.

I'm not seeing any option in the PR to enable that, but I'm pretty sure I've seen that before on other PRs. I'm not sure what I'm missing here. Maybe because my source branch is in an organization?

> To create the rename - add the change to https://github.com/Homebrew/homebrew-cask/blob/master/cask_renames.json

Will do, thanks.

rtharston commented:
@bevanjkay, we released a new version this morning so I updated the version to that. While looking at the documentation to better understand how version updates work I found `auto_updates` and the `sparkle` strategy for `livecheck` so I added those as well.

Using Sparkle then highlighted that the cask didn't have the macOS version dependency so I added that as well.

Originally I just copied everything from the remotion cask which someone else made for us, but I'm glad I found the documentation and added the extra information.

One question though, we support 11.3 and up, not 11.0. I couldn't find a way to check for 11.3 instead of :big_sur. Are you aware of a way to check for minor versions?

miccal commented on Casks/m/multiapp.rb:
```diff
@@ -0,0 +1,25 @@
+    strategy :sparkle, &:short_version
+  end
+
+  auto_updates
```
```suggestion
  auto_updates true
```

rtharston replied:
I was wondering if something was needed here, but none of the `audit` operations complained. 🤷‍♂️

miccal commented on cask_renames.json:
```diff
@@ -1,5 +1,6 @@
   "ankerslicer": "ankermake",
   "cloudapp": "zight",
-  "sublime-text3": "sublime-text"
+  "sublime-text3": "sublime-text",
+  "remotion": "multiapp"
```
```suggestion
  "remotion": "multiapp",
  "sublime-text3": "sublime-text"
```

miccal commented:
> One question though, we support 11.3 and up, not 11.0. I couldn't find a way to check for 11.3 instead of :big_sur. Are you aware of a way to check for minor versions?

No, we can only check major versions.

rtharston commented:
@miccal, thanks for the review notes! I've fixed both.

miccal commented:
Thank you @rtharston.
